subscription_manager_test.go

```diff
@@ -569,3 +569,13 @@ func TestProcessSubscribePayloadCipherErr(t *testing.T) {
 	<-done
 	//pn.Destroy()
 }
+
+func TestDecryptionProcessOnNoEncryptedMessage(t *testing.T) {
+    assert := assert.New(t)
+    pn := NewPubNub(NewDemoConfig())
+
+    result, err := parseCipherInterface("test", pn.Config, pn.getCryptoModule())
+
+    assert.Nil(err)
+    assert.Equal("test", result)
+}
```

Comment:
I am not sure if it is worth adding it here but in Kotlin there are tests(one parameterized) that checks behaviour of the sdk when you try decrypt unencrypted message.

For this message: "this is unencrypted message"	
This error is thrown:
PubNubException(errorMessage=Input length must be multiple of 16 when decrypting with padded cipher, pubnubError=PubNubError(name=CRYPTO_ERROR, code=135, message='Error while encrypting/decrypting message. Please contact support with error details.'), jso=null, statusCode=0, affectedCall=null)

For this message: “this is not encrypted message”
This error is thrown:
`java.lang.IllegalArgumentException: bad base-64`

For user they are handled the same way. The thing is that initially we didn't handle last example properly because this is not PubNub exception but `java.lang.IllegalArgumentException`

Reply:
Maybe there is currently lack of my knowladge how to prepare good test cases in Go. I wanted to have **any** test if the function behaves as desired. 

Right now I'm trying to catch any `base64` error that means it couldn't decode that - what indicates that message is not encrypted or encrypted in other way than using our SDKs.